Gruntfile.js

```diff
@@ -253,10 +259,12 @@ module.exports = function(grunt) {
                     '<%= meta.metaJsFiles %>',
                     'site/src/**/*'
                 ],
-                tasks: ['site']
+                tasks: ['site'],
+                options: {
+                    livereload: true
```

Comment:
yep can do watch:components and site:serve together now. Probably with a little tuning could work a way to get watch:components to kick site rebuild but didnt have in this pass.

Will specify port for clarity